### ISSUE TITLE
Output RTSP stream

### ISSUE THREAD

**Issue by mikepcw** [leswright1977/PyThermalCamera#14]
Would it be possible to either to render to screen AND output the same to an RTSP stream, or have the option to display either on screen OR output to RTSP?
I would like to ingest the RTSP stream into my frigate for recording.

**Comment by ttt-repl** [comment]
Agreed! I'm grateful for what we have so far but I'd love the ability to output my thermal imaging to RTSP for viewing on other devices too.

**Comment by Fabelwesen** [comment]
+1

**Comment by jm101093** [comment]
https://github.com/jm101093/Thermal_camera_TOPDON_TC001_rtsp_server

made a quick RTSP server if you guys are still looking.
